_docs/platforms/kubernetes/install/1.10.md

```diff
@@ -9,44 +9,62 @@ redirect_from: /docs/install/kubernetes
 
 # Kubernetes 1.10+
 
+{% include operator/install.md %}
+
+## Custom Installations
+
+There is a variety of flavours, versions and particularities in the container orchestrator scope.
+Because of that, StorageOS installation procedures aims to be flexible so it can fit different needs
```

Comment:
Because of this, StorageOS installation procedures aim to be flexible...

Reply:
done